### PR TITLE
"Distance travelled by mode" section - layout for icons section

### PR DESCRIPTION
**Pull request by RCode-Blue**
## Overview of the Pull Request:
Mobile layout for travel method icons in "Distance travelled by mode" section

Fixes to the formatting of the text and icons header in the _Distance travelled by mode of transport_ section of the _Results_ page

## How Has This Been Tested?
Layout for icons and text should display as follows:

Desktop view:
image

Mobile view:
image

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [X] Is your pull request up-to-date with `main` branch?
- [X] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [X] Have you written instructions on how to test that your changes work as expected?
- [X] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx [hunk 1]
```diff
@@ -27,21 +27,21 @@ export default function DistanceTravelledMode({ data }) {
...
+      <Flex direction={["column", "row"]} width="100%" my="15px">
         <Flex
           direction="column"
           width={["100%", "50%"]}
           align={["left", "center"]}
+          justifyContent="space-between"
         >
           {/* row for the sub title */}
-          <Flex direction="row">
+          <Flex direction="row" alignItems={"center"} my="5px">
```
Comment (OlaShabalina) [inline components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx:38]: `alignItems` is `align` in Chakra, no need `{` here 

```suggestion
          <Flex direction="row" align="center" my="5px">
```

File: components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx [hunk 2]
```diff
@@ -71,14 +71,14 @@ export default function DistanceTravelledMode({ data }) {
         {/*Active,Public/Shared/Method right column */}
 
         <Flex direction="column" width={["100%", "50%"]}>
-          <Flex direction="row">
+          <Flex direction="row" alignItems={"flex-start"} my="5px">
```
Comment (OlaShabalina) [inline components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx:74]:
```suggestion
          <Flex direction="row" align="flex-start" my="5px">
```

**Review by OlaShabalina — CHANGES_REQUESTED** [review]
Great work @RCode-Blue , just a few `align` attributes to correct.

**Review by OlaShabalina — APPROVED** [review]

**Review by hqtan — APPROVED** [review]
luv it! ❤️ ❤️ 

thx @RCode-Blue ! 🙇🏽

**Pull request merged**